designs/Prerelease-Option.md

```diff
@@ -0,0 +1,69 @@
+# Spec Name
+
+* Status: In Review
+* Author(s): [Martin Ruiz](https://github.com/martinrrm)
+* Issue: [4699](https://github.com/NuGet/Home/issues/4699) dotnet add package - support for NuGet "Pre" version
+
+## Problem Background
+
+Customers want to add the latest version of a package to their project, even if the latest version is prerelease.
+
+## Who are the customers
+
+All dotnet.exe customers
+
+## Goals
+
+Add an option to the command `add package` to enable installing latest prerelease version.
```

Comment:
Just nitpick, but I think the goal should be framed as helping a customer accomplish a "job to be done." In this case the goal should be something like:

 "Allow dotnet CLI users to easily install the latest version of NuGet package, including prereleases. This can be accomplished by adding an option to the existing `add package` that enables installing prerelease versions."